packages/sdk-ts/src/client/indexer/rest/IndexerRestExplorerApi.ts

```diff
@@ -627,4 +629,55 @@ export class IndexerRestExplorerApi extends BaseRestConsumer {
       })
     }
   }
+
+  async fetchCommunitySpendPool(params: {
```

Comment:
Nope, we should have `fetchBankTransfers` and then pass the `is_community_pool_related` as a parameter on the function call.

Reply:
I've resolved it.